server/src/main/java/com/objectcomputing/checkins/services/onboardeeprofile/OnboardingProfile.java

```diff
@@ -0,0 +1,267 @@
+package com.objectcomputing.checkins.services.onboardeeprofile;
```

Comment:
All the /onboardeeprofile/* files have to be moved into /onboard-server directory and integrated into the new backend app.

Reply:
Done